### ISSUE TITLE
Updating script to Python3

### ISSUE THREAD

**Issue by bbhtt** [davidxk/SolitaireBot#1]
I updated the script to follow Python3 conventions also had to change some things like Queue to queue etc. However stuck here:
```
$ python3 shenzhen-solitaire.py
Traceback (most recent call last):
  File "shenzhen-solitaire.py", line 15, in <module>
    import pyscreenshot
  File "/home/korte/.local/lib/python3.8/site-packages/pyscreenshot/__init__.py", line 5, in <module>
    from pyscreenshot.loader import Loader, FailedBackendError
  File "/home/korte/.local/lib/python3.8/site-packages/pyscreenshot/loader.py", line 2, in <module>
    from pyscreenshot import plugins
  File "/home/korte/.local/lib/python3.8/site-packages/pyscreenshot/plugins/__init__.py", line 5, in <module>
    from pyscreenshot.plugins import wxscreen, gtkpixbuf, qtgrabwindow, scrot, \
  File "/home/korte/.local/lib/python3.8/site-packages/pyscreenshot/plugins/scrot.py", line 2, in <module>
    from easyprocess import extract_version
ImportError: cannot import name 'extract_version' from 'easyprocess' (/home/korte/.local/lib/python3.8/site-packages/easyprocess/__init__.py)
```
Reason for doing so is Linux distros are no longer going to support Python2 from this year.

**Comment by fatestigma** [comment]
It seems like an error in third-party library. I noticed that David froze  all dependencies' version in requirements.txt. Could you check if upgrading to latest version helps? Also, could you provide some details about your environment(Python version, and library versions). Thanks.

**Comment by bbhtt** [comment]
```
Python 3.8.2
PyAutoGUI==0.9.33
pyscreenshot==0.4.2
Pillow==3.1.2
```
OS is Ubuntu 20.04 Xfce, might try to run on an arch VM later. I think I have an older kernel with Python2.

I tried upgrading the requirements, skips upgrade for EasyProcess,pyscreeze,PyTweening (Requirement already satisfied, skipping upgrade); rest are up-to-date.

Edit. Fixed it, had to remove the modules,clean and manually upgrade each one. I need to fix more of the code still. Closing this!
